src/index.ts

```diff
@@ -299,6 +302,19 @@ if (!window.customElements.get('md-strikethrough')) {
   window.customElements.define('md-strikethrough', MarkdownStrikethroughButtonElement)
 }
 
+class MarkdownTableButtonElement extends MarkdownButtonElement {
+  constructor() {
+    super()
+    const tableMarkdown = '| header | header |\n| ------ | ------ |\n| cell | cell |\n| cell | cell |\n'
```

Comment:
What if a user wants to create a table with three or more columns?

Reply:
This part add a Minimum to start a table. You can add more column like github's markdown do that.

Example:

3 columns

'| header | header | header |
| ------ | ------ | ------ |
| cell | cell | cell |
| cell | cell | cell |'

| header | header | header |
| ------ | ------ | ------ |
| cell | cell | cell |
| cell | cell | cell |

3 lines

'| header | 
| ------ | 
| cell | 
| cell | 
| cell | '

| header | 
| ------ | 
| cell | 
| cell | 
| cell |